app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
```

Comment:
Since you are using "file.txt" as a fallback for `filename`, you can already declare the filename with this name and change it later if the user inputs a name to it.

```suggestion
filename = "file.txt"
```